source/wp-content/themes/wporg-news-2021/sass/components/_local-header.scss

```diff
@@ -239,7 +234,9 @@
 	}
 
 	a {
-		color: var(--bar-link-color);
+		// This important is needed to override the navigation
+		// block link color.
+		color: var(--bar-link-color) !important;
```

Comment:
It's also overriding the hover color below

Reply:
This was more a note for myself. I've removed the need for the `!important` with a more specific selector.